push/model/src/main/java/org/jboss/aerogear/unifiedpush/message/apns/APNs.java

```diff
@@ -111,6 +117,11 @@ public void setContentAvailable(boolean contentAvailable) {
 
     public void setLocalizedTitleArguments(String[] localizedTitleArguments) {this.localizedTitleArguments = localizedTitleArguments;}
 
+	public String[] getLocalizedArguments() { return localizedArguments; }
+	
+	public void setLocalizedArguments(String[] localizedArguments){ this.localizedArguments = localizedArguments; }
 
-
+	public String getLocalizedKey() { return localizedKey; }
+	
+	public void setLocalizedKey(String localizedKey) { this.localizedKey = localizedKey; }
```

Comment:
again, wrong indentation + please avoid one-liners (keeping the same formatting as the code around is usually best approach)